containers/Connector/Connector.tsx

```diff
@@ -70,12 +72,14 @@ const useConnector = () => {
 	}, [signer, dispatch]);
 
 	useEffect(() => {
-		const rateInterval = setInterval(() => {
-			dispatch(setExchangeRates());
-		}, 15000);
+		const ratesListener = (exchangeRates: Rates) => {
+			dispatch(setExchangeRates(exchangeRates));
+		};
+
+		sdk.events.on('exchangeRates_updated', ratesListener);
 
 		return () => {
-			clearInterval(rateInterval);
+			sdk.events.removeListener('exchangeRates_updated', ratesListener);
```

Comment:
same here so we're not referencing events directly but something like `sdk.exchange.removeListener(ratesListener)`

Reply:
Good idea.

Reply:
Added a simple `removeRatesListeners` method.